docs/csharp/programming-guide/interop/walkthrough-office-programming.md

```diff
@@ -43,7 +43,11 @@ Both languages enable embedding of type information, which allows deployment of
 This walkthrough demonstrates these features in the context of Office programming, but many of these features are also useful in general programming. In the walkthrough, you use an Excel Add-in application to create an Excel workbook. Next, you create a Word document that contains a link to the workbook. Finally, you see how to enable and disable the PIA dependency.  
   
 ## Prerequisites  
+<<<<<<< HEAD
 You must have Microsoft Office Excel and Microsoft Office Word installed on your computer to complete this walkthrough.  
+=======
+ You must have Microsoft Office Excel 2007 or later and Microsoft Office Word 2007 or later installed on your computer to complete this walkthrough.  
+>>>>>>> Updates
   
```

Comment:
@mairaw This PR thus far keeps it, but it renames it from ...
```
docs/visual-basic/getting-started/breaking-changes-in-visual-studio-2015.md
```
to
```
docs/visual-basic/getting-started/breaking-changes-in-visual-studio.md
```
What we'd be missing is a similar doc ...
```
docs/csharp/getting-started/breaking-changes-in-visual-studio.md
```
... over in the csharp area.

At least today, this doesn't seem to provide any useful content. There are no "breaking changes" to show AFAICT from its current content. Should they be kept around in case in the future there are language-related VS breaking changes to report?

Four options:
1. Keep the doc but also keep the 2015 name on it
2. Keep the doc and rename it as done in the PR to remove the year
3. Keep the doc (either 1 or 2 above) and mirror it over in the csharp area
4. Kill this doc off :gun: